### PR TITLE
Added instance in content/v3/troubleshooting.md

### PR DESCRIPTION
**Pull request by CalderWhite**
I added an instance I ran into many times, when making requests to the api in python 3.

### CODE REVIEW

File: content/v3/troubleshooting.md [hunk 1]
```diff
@@ -44,5 +44,14 @@ should be prepared to answer technical questions about your goal and your planne
 dreams with the current rate limit (but don't worry, we'll help you out).
 
 {% endif %}
+## Why does the api always return "Problems parsing JSON"?
```
Comment (Shredder121) [inline content/v3/troubleshooting.md]: Could you add a blank line here, for consistency?

File: content/v3/troubleshooting.md [hunk 2]
```diff
@@ -44,5 +44,14 @@ should be prepared to answer technical questions about your goal and your planne
...
+when doing a PUT request to the GitHub api (specifically repository contents)    
+the request must be in `https`, not `http`. If it is in http then the api will always return
```
Comment (Shredder121) [inline content/v3/troubleshooting.md]: Trailing whitespace here.

**Comment by 4457** [comment]
458ae32b2e2326549fee991c98d39db1d8fed427

**Comment by 4457** [comment]
xawxaw118884@icloud.com

**Comment by MikeMcQuaid** [comment]
Passing on this as it's implied throughout the documentation but thanks for the PR anyway!

**Pull request closed without merging**